### ISSUE TITLE
Search for new lenses in KIDS and KABS

### ISSUE THREAD

**Issue by KiaraSpin** [nrnapolita/KIWI_LSST#11]
Chiara is working on this with A.Agnello at ESO. The goal is to build up a sample of promising candidates to submit for a Spectroscopic Follow up proposal (Large Program?)

**Comment by KiaraSpin** [comment]
GM 29/01 
We are starting to prepare a "method paper" highlight the different ways we used to select candidates and in general to "search" for lenses in all-sky surveys. 
Currently we have already "successfully" applied 3 different method and we are investigating the complementarity of them. 

AI Fedor -> provide Chiara with the images and cutoff of newest candidates from meth3

**Comment by KiaraSpin** [comment]
Fedor provided the cutoff of the new candidates. The visual inspection is almost concluded. 
Nicola and Chiara checked all the images and made a lists of candidates (separately). 
AI Chiara -> compare the lists and grade the candidates.

Also, Alexey provided images of ALL the candidates collected so far with gradfe >1.5 where he subtracted the sources. This is very useful to check for the presence of the deflector. 
AI Chiara->  once terminating with visual inspection and grading, make a FINAL list of candidate for each method
We need to be ready with the proposal by the end of march.

**Comment by KiaraSpin** [comment]
GM 19/03
UPDATE: two of the best candidates were observed by Crescenzo at TNG. 
We need to set up a strategy to reduce the data. 

Chiara (with A.Agnello) is preparing a X-Shooter proposal. 

Chiara is writing the "method paper" and has already circulated a draft to collaborators. 
Figures are still missing, the cross-match with 2QZ could go into the paper, as well as the spectroscopic confirmation ... if we manage...

AI @nrnapolita : Cross match spectroscopic surveys with KiDS Complete catalog (thor)

**Comment by KiaraSpin** [comment]
GM 14/5 
Method paper sent to KiDS collaboration. A window of two weeks has been given for comments.

**Comment by KiaraSpin** [comment]
GM 18/06
Paper I has been submitted. Now, finger crossed and let's wait for ref. report.
Meanwhile, Chiara is starting to work on the KABS paper.
@nrnapolita Check the overleaf draft that Chiara sent

**Comment by KiaraSpin** [comment]
GM 02/07 
@ctortora Check the situation with Enrico on KABS-Arcs! Access the status of the data uploaded by Fedor. 

We got the referee report for Paper I and for Agnello2018, Chiara is working on it.

**Comment by KiaraSpin** [comment]
GM 09/07
ARCS in KABS: Enrico has no time to download and run the code. BUT this is low priority for him!
QSOs in KABS: New selections based on WISE+KABS. Chiara and Nicola are working on final list of candidates


REFEREE REPORT: Chiara finished the ref report. She only needs to clarify one point with Adriano. She will send a reminder to him (again)

**Comment by KiaraSpin** [comment]
GM 18/07 
Paper I (Spiniello et al. 2018) has been accepted :) 👍

**Comment by KiaraSpin** [comment]
GM 26/07 

**AI Chiara**: Contact Timo to receive a summary of the july NTT run

**AI Chiara**: Produce a list of ONLY NEW QSOs candidates from KABS and send it to Fedor to produce the cutouts. 

**AI Nicola**: Check availability of 4 bands catalog in KABS

**AI Crescenzo**: We need to "complete" the Arcs part of the method paper. Enrico is really busy finishing the thesis but at least we can ask Enrico "how much" he already inspected

**Comment by KiaraSpin** [comment]
GM 10/09

QSOs: the procedures have been set. Chiara produced a final list of candidates and Fedor produced the cutouts. Now visual inspection is necessary. 
Many people in the team involved (Diana, Peppe, Nicola, Crescenzo, Mario). 
**Deadline 21/09**

ARCs: Enrico is very busy with finalizing the thesis.

**Comment by KiaraSpin** [comment]
GM 15/10 
Dea all, 
I have created a google spreadsheet with the list of candidates. 

 You find it [HERE](https://docs.google.com/spreadsheets/d/1N9MXgiozV9MG_KwU2lNp8UyqGkqoijr--6_ntKjLCUE/edit?usp=sharing) but you need to ask my permission to access it. 

In the sheet, you will find find different TABS (down, left corner) with the different lists of objects.
I have divided them into: 

**DOUBLE COLOUR MATCHES**: where two nearby point-like source have passed the NIR&OPT color selection 

**OTHERS**: these are multiple objects (always within 5") where *one* of them only passed the selection.

 **ARCS**: here there are the galaxy-galaxy lenses candidated found by the color method AND the CNN from Enrico. There are another two tabs for the ARCS, where I divided them into two groups according to who found them/how they were found. 

Finally, I have added a TAB called 
**OLD SELECTION** which has few systems that were there in my previous color selection and are not there now (I change slightly the NIR color constraints, nevertheless we can consider them as candidates)


For these of you that still did not send me the grading, and are still interesting in this, please simply insert it in the corresponding column.

**Comment by KiaraSpin** [comment]
AI Chiara: set a deadline for visual inspection and share the Overleaf paper draft with Crescenzo and Nicola

**Comment by KiaraSpin** [comment]
GM 03/12
AI Chiara: re-start the "paper-flow" for the KABS-Lens. Coordinate with Enrico&Crescenzo.

**Comment by KiaraSpin** [comment]
GM 17/12
AI @nrnapolita  and Crescenzo: 
Work on the draft of the lens paper

**Comment by KiaraSpin** [comment]
GM 21/01
Same as previous meeting. AI @nrnapolita and @ctortora Work on the draft

**Comment by KiaraSpin** [comment]
Update: the KABS-Lens paper has been submitted. The Open action items are closed, until we receive the referee report

**Comment by KiaraSpin** [comment]
KABS-Lens paper has been rejected. 

Attached here the referee's response: 


Dear Dr Napolitano,
Copied below are the comments on your manuscript entitled "KABS-GraL: The KABS strong Gravitational Lensing project", ref. MN-19-1678-MJ, which you submitted to Monthly Notices of the Royal Astronomical Society.
**The reviewer has recommended against publication in MNRAS. In view of this report, we suggest you withdraw this particular submission. If you decide to withdraw the manuscript, please let me know.**




**Scientific Editor's Comments:**
Editor
Comments to the Author:
The referee has recommended Rejection of this paper on the grounds that the analysis and description do not meet the rigorous standards required for publication in MNRAS. As the referee has proposed a number of ways to improve the paper, and has indicated they are willing to review it again, I have moderated this to Withdrawal to allow you time to review and revise the paper, hopefully with a view to re-submitting once the paper has substantially improved.



**Reviewer's Comments:
Reviewer: 1**

Comments to the Author
'KABS-GraL: The KABS strong Gravitational Lensing project' presents the first results from a gravitational-lens search over ~120-250 square degrees of ground-based imaging data from the KiDS-ATLAS Bridging Survey. The manuscript is a continuation of similar searches by the authors, which is problematic in several respects. First, a smaller area with less wavelength coverage is investigated than their previous work. Furthermore, the methods used have been taken from those previous papers and only partially adapted to the current dataset. Finally, spectra for only one candidate have been presented in spite of the manuscript noting that more time has been awarded. On these grounds, with no substantially new methods, datasets, or results presented, the current manuscript is not ready for publication in MNRAS.

However, an even more critical issue is that simple checks of the reliability of the searches presented here have not been performed, and the catalog has clear false positives. A quick comparison with the Gaia DR2 proper motion and parallax data shows that at least one candidate lensed-QSO image is a star for each of the 5th through 14th most highly ranked objects (excluding the non-QSO candidate KABS1203-3738 and the mislabelled object KABS1423-3147 but including the mislabelled object KABS1919-4036). This search also rules out KABS1922-4052 (the claimed lensing galaxy is a star), KABS1223-3618 and KABS1447-3102 -- a total of 11 candidates and nearly all of the QSO candidates.

An extremely troubling aspect of this is that the images in Figure 3 for KABS1423-3147 and KABS1919-4036 do not correspond to the RA and Dec given in Table 1, the second of which point to a pair of stars also ruled out by Gaia data (and the label should be 1917-4055). I have not checked all of the objects, but this is incredibly worrying.

I therefore must recommend against publication of this manuscript in MNRAS. A more complete analysis of the candidates must be performed, including verifying that potential QSO images are not stars (or extended objects). Any future submission should apply the analysis of Section 4.1 to all of the QSO lens candidates to demonstrate point-like morphologies, similar colors and the possible presence of a lensing galaxy. The arc candidates should be fit with lens models to demonstrate that the arc morphologies are consistent with gravitational lensing. Every precaution must be taken to present the catalog data accurately to ensure the catalog is useful and not misleading. Finally, the authors should wait to publish a full list of candidates from the completed KABS survey or at a minimum wait until they have obtained more spectra to evaluate how well their methods work with KABS data.

I hope the authors will address these concerns, and in anticipation of that I have provided more specific comments and questions about the manuscript below.


Introduction:

The first set of references (Marshall et al. 2007; Shu et al. 2016; Bellagamba, Tessore, & Metcalf 2017) do not discuss lensed quasars. Conversely, when discussing galaxy-galaxy strong lenses in the third paragraph the authors have included 4 papers that relate to galaxy-QSO strong lenses (Mao & Schneider; Dalal & Kochanek; Nierenberg et al; Gilman et al). This confusion between galaxy-galaxy and galaxy-QSO lenses arises again when citing Jacobs et al. (2019) for using machine learning and CNNs to find 'the more rare lensed QSOs' (that paper was about galaxy-galaxy lenses; also note that Ostrovski et al and Krone-Martins did not use 'convolutional networks').

The fourth paragraph of the introduction states that the biggest limitation to strong-lensing science is the 'paucity of confirmed lenses' -- can the authors give examples of how the ~200 confirmed galaxy-galaxy lenses and ~200 confirmed galaxy-QSO lenses are not enough?

Large surveys have found many new lenses, but several of the cited papers have reported candidates rather than confirmed lenses, which the authors should clarify (since the previous paragraph stressed the need for confirmed lenses). Also note that More et al. 2016, Williams et al. 2018 and Lemon et al. 2019 (all cited elsewhere) have reported many new QSO lenses, while More et al. 2016 and Sonnenfeld et al. 2018 (both not cited) have reported galaxy-galaxy lens candidates.


Section 2:

It is not clear to me why the goals of KABS needed to be enumerated in this manuscript.

How were the final photometric catalogs created?

The details about KiDS could be provided in the first paragraph of Section 2 to remove the redundant wording.

I do not understand the caption of Figure 1. What are the pink-shaded areas? What do the blue contours mean?


Section 3:

From a ground-based survey perspective, what is the difference between 'good seeing' and 'great spatial resolution'?

What are the implications of the survey depth in g and r, given the gri-color selection, on the expected number of lensed QSOs?

The manuscript claims that P19a simulated 'multiple images of quasars' but I find no mention of that in P19a.

Is the source-lens contrast that was applied to the training set for r-band images also appropriate for the i-band images used here?

Why is the galaxy-galaxy lens search limited to objects with i < 20.5? Is this more generous than the KiDS cut, ie what are the typical r-i colors?

Is the grade G the median or the mean of the 5 grades? I had assumed the grading was done on an integer basis, but this cannot be true given the numbers in Tables 1, 5, 6 and 7. Did all of the graders use non-integer grades, or did some assume an integer-based scheme (in which case the grades are heterogeneous)? Were the quality of the grades assessed using objects from the CNN training set? How many objects were given a grade of 4 by any grader? Why was the criterion G>=2 chosen?

The WISE color-selection criteria in Equation 1 need to have improved punctuation to properly express the logic around the 'OR' statements. Footnote 6 does not state that W3 errors are required, so does this mean upper limits on W3 are used? What is the reason for loosening the S18 criteria to (W3 > 11) AND (W2-W3 > 0), since very few known lenses are added but many more contaminants are also included?

Finet et al. (2015) also presented an AGN+multiplets lens search algorithm, using X-ray-selected sources instead of WISE.

What does the distribution of the 171850 separations look like? If the mean separation is 0.6" then the median is presumably even smaller. Is it expected that KABS, with typical seeing 0.8", would deblend and catalog such close objects? Are the smaller-separation objects clustered (for example, in regions that had particularly good seeing)?

What photometry was used to put the known lenses on Figure 2? This figure might be clearer if contours are used for the stars and the KABS+WISE objects. Lemon et al. (2019) compiled a list of 204 known quasars, presumably including the subset presented here.

How many instances of the 'single colour match' objects also include at least one object within the selection radius but outside of the color box from Figure 2?

As mentioned earlier, photometry should be performed for all of the lensed QSO candidates in Table 1/Figure 3 (applying the procedure from Section 4.1), and lens models should be created for the candidates with possible Einstein arcs to show they are consistent with lensing features, since the KABS data might be the best imaging available for these lenses for a long time.

A cross-reference with the Gaia DR2 catalog finds many of the QSO-lens candidates actually consist of at least one star (with significant proper motions and or parallaxes). The only mutliplet-only objects from Table 1 that remain are KABS1233-3542 (the confirmed lens), KABS1423-3147 and KABS1058-3440 which does not have a counterpart in Gaia DR2. What is the distribution of local stellar densities for these objects?

The name for KABS1050-2441 also does not correspond to the Right Ascension and Declination given in Table 1, which give KABS1050-3441. What is the correct RA and Dec of the object KABS1919-4036 shown in Figure 3? What was the grade assigned to the object cataloged as KABS1919-4036 in Table 1 (which is really 1917-4055)? Likewise a VHS image (and Gaia search) for KABS1423-3147 shows that the image in Figure 3 is clearly not correct for this object. What object is the imaging showing?


Section 4:

Also as mentioned earlier, this manuscript would be much stronger presenting the results from a more complete spectroscopic campaign.

Why is Gaia photometry used instead of the photometry described in Section 2?

Figure 4 should also show the total residuals, including having subtracted the lensing galaxy (as in Sergeyev et al. 2018). What is the cause of the significant color gradient in the lower-right panel?

The astrometry is described as precise, so the errors on the QSO separation and B-G separation should be stated.

The A and B spectra are not identical, for example the strong absorption seen redward of SiV in A but not B. What is meant by 'fitting' the emission lines? How much has B been shifted by in Figure 5?

How does the spectroscopic incompleteness of SDSS affect the deflector redshift? The median and 68% region of the points from Figure 6 are more appropriate to quote than the mean and standard deviation.

I find Figure 7 to be confusing. A single large two-column figure with no spectral breaks would be better, with the (scaled) sky also shown. Why does the 4500-4700 angstrom region look different between the top and bottom panels? Please discuss the presence of the z = 0.79 absorption features in the spectrum of A but not B, particularly since B is closer to the photometric position of the lensing galaxy.

What is the mass-to-light ratio that is inferred if one assumes z = 0.21 or z = 0.79? Which redshift is more likely in that case?

What is meant by no 'such evidence from the spectral analysis' for microlensing? The timescales for microlensing can be quite long and the observed fluxes in both the photometry and the spectra could be affected by microlensing. Which fluxes were used, the photometric (in that case, which band) or spectroscopic? What errors were used, considering the offsets between the photometric bands and spectra?

The lensing model has 8 parameters (5 for the lensing mass and 3 for the source position and flux) compared to 6 constraints. This should be mentioned, and how were the errors in Table 4 calculated?


Section 5:

In what sense was the CNN 'customized for KABS'? This seems to contradict Section 3.1.

In what sense does presenting a list of high graded candidates 'demonstrate the validity of the methods'? In fact, cross-checking with Gaia apparently demonstrates the opposite.

Does Figure 6 disfavor the z = 0.79 solution if the spectroscopic completeness of SDSS is considered?

According to Section 3.1, the CNN produced 235 candidates, not 64 as quoted multiple times.

A set of input color selection criteria were used to select potential candidates. Where do the possible lenses presented in Tables 1, 5, 6 and 7 lie in this color space (does the visual inspection step select a certain class of objects, for example)?



Other comments:

The manuscript has substantial language problems and must be proofread more carefully. For example, two sentences from Section 3.1: However, for the time being, this is the most secure approach to ensure a large training set, with physical noise and seeing as much similar as possible to real data, but on the same time have a number of objects large enough to provide a proper training to the CNN. When KABS will be completed, and a larger area will be covered, we will be able to define a large enough LRGs sample of galaxies observed by KABS in the i-band to build a more appropriate mock training set.

good spatial resolution -- good spatial sampling

We will provide more details on future improvements in Section 4 -- should be Section 5.

The symbol G is used in three different ways: the grade, the lensing galaxy, and the Gaia bandpass.

The text uses WISE-KABS but Figure 2 calls them KABS+WISE.

GAIA -- Gaia (and KIDS -- KiDS)

The top row of Figure 4 -- Figure 4 (the i-band is on the lower row, for example).
